src/modules/fancyzones/lib/Zone.cpp

```diff
@@ -105,7 +105,7 @@ void Zone::SizeWindowToZone(HWND window, HWND zoneWindow) noexcept
         }
     }
 
-    if ((::GetWindowLong(window, GWL_EXSTYLE) & SWP_NOSIZE) != 0)
+    if ((::GetWindowLong(window, GWL_STYLE) & WS_SIZEBOX) == 0)
```

Comment:
Would it make sense to also check for `SWP_NOSIZE` and not resize if at least one of the two checks will be positive?

Reply:
Probably not. That would mean that there is a window that have SWP_NOSIZE property, yet still could be resized manulaly.
Fancy zones wouldn't resize it on snapping, but user could resize it later or earlier

Reply:
OK.